docs/usage.md

```diff
@@ -6,61 +6,162 @@
+| • Tumor only RNA                    | :white_check_mark: |                     -                     |
+
+<sub><sup>\*</sup> Supported analyses relate to the TSO500 panel only</sub>
+
+## Samplesheet
+
+A samplesheet that contains information of each input in CSV format is needed to run oncoanalyser. The required input
+details and columns are [described below](#column-descriptions).
+
+Several different input filetypes beyond FASTQ and BAM are recognised, including intermediate output files generated
+during execution such as the PURPLE output directory. The full list of recognised input filetypes is available
+[here](https://github.com/nf-core/oncoanalyser/blob/1.0.0/lib/Constants.groovy#L58-L90).
+
+### Simple example
+
+#### FASTQ
+
+:::note
+
+Currently only non-interleaved paired-end reads are accepted as FASTQ input.
+
+:::
+
+```csv title="samplesheet.csv"
+group_id,subject_id,sample_id,sample_type,sequence_type,filetype,info,filepath
+P1__wgts,P1,SA,normal,dna,fastq,library_id:SA_library;lane:001,/path/to/P1.SA.normal.dna.wgs.001.R1.fastq.gz;/path/to/P1.SA.normal.dna.wgs.001.R2.fastq.gz
+P1__wgts,P1,SB,tumor,dna,fastq,library_id:SB_library;lane:001,/path/to/P1.SB.tumor.dna.wgs.001.R1.fastq.gz;/path/to/P1.SB.tumor.dna.wgs.001.R2.fastq.gz
+P1__wgts,P1,SC,tumor,rna,fastq,library_id:SC_library;lane:001,/path/to/P1.SC.tumor.rna.wts.001.R1.fastq.gz;/path/to/P1.SC.tumor.rna.wts.001.R2.fastq.gz
 ```
 
-### Multiple runs of the same sample
+#### BAM
+
+:::note
+
+Inputs with the `bam` filetype will be processed by MarkDups as required by hmftools. Where an input BAM has already
+been processed specifically by [HMF
+MarkDups](https://github.com/hartwigmedical/hmftools/blob/mark-dups-v1.1.7/mark-dups/README.md), you can avoid needless
+reprocessing by setting `bam_markdups` as the filetype instead. It is important to understand that duplicate marking by
+other tools (e.g. GATK) cannot be used as a substitute since HMF MarkDups performs key operations beyond just duplicate
+marking.
+
+<br />
 
-The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
+Please note there are other essential requirements around the use of BAMs as inputs, see the warning above in the
+[Introduction](#introduction).
 
-```console
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz
+:::
+
+```csv title="samplesheet.csv"
+group_id,subject_id,sample_id,sample_type,sequence_type,filetype,filepath
+P1__wgts,P1,SA,normal,dna,bam,/path/to/P1.SA.normal.dna.wgs.bam
```

Comment:
Again the double underscores, why?

Reply:
See https://github.com/nf-core/oncoanalyser/pull/29#discussion_r1666195902

Reply:
Also marking this as resolved given overlap of https://github.com/nf-core/oncoanalyser/pull/29#discussion_r1665365857